### DISCUSSION TITLE
Not able to debug the problem in brownie run scripts/deploy.py

### DISCUSSION THREAD

**Discussion by Adi9876** [smartcontractkit/full-blockchain-solidity-course-py#1879] [Q&A]
deploy.py only contains print statement for testing purpose 

<img width="730" alt="Screenshot 2023-07-02 at 3 41 45 AM" src="https://github.com/smartcontractkit/full-blockchain-solidity-course-py/assets/62845979/fd441597-cc52-4a18-a2fa-d0f8293c1f39">

**Comment by kapasainitishreddy** [comment]
Try using ganache cli

**Reply by Adi9876** [reply]
it says ganache-cli depreciated and so it uses ganache
